### PR TITLE
Add 1.16.1 support

### PR DESCRIPTION
**Pull request by Heath123**
1.16.1 and 1.16 should be the the same. I couldn't make the dependencies work to test this though. examples/server_world/mc.js requires a version of prismarine-chunk that doesn't exist but that was an issue before this too.

### CODE REVIEW

File: src/version.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict'
 
 module.exports = {
-  defaultVersion: '1.16',
-  supportedVersions: ['1.7', '1.8', '1.9', '1.10', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16']
+  defaultVersion: '1.16.1',
+  supportedVersions: ['1.7', '1.8', '1.9', '1.10', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16', '1.16.1']
```
Comment (Saiv46) [inline src/version.js:5]: Major version duplication (1.16 and 1.16.1) - you just needed to replace `'1.16'` with `'1.16.1'`

Comment (Heath123) [reply src/version.js]: Fixed

**Comment by Saiv46** [comment]
`node-minecraft-data` update with 1.16.1 not released yet

**Comment by rom1504** [comment]
Ah sorry just saw this PR now, I already did a similar change in master, we have 1.16.1 support

**Pull request closed without merging**